### PR TITLE
05_motseacciona.md

### PR DESCRIPTION
**Pull request by montseacciona**
- [x] Added the username
- [ ] Passed Test

### CODE REVIEW

File: _slides/05-montseacciona.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (unaizorrilla) [inline _slides/05-montseacciona.md:3]: Woul be good to remove latest '!' character

File: _slides/05-montseacciona.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+Hello world
```
Comment (unaizorrilla) [inline _slides/05-montseacciona.md:6]: The idea is use this line to show up your nickname
```suggestion
Hi I'm montseacciona
```

**Review by unaizorrilla — CHANGES_REQUESTED** [review]
LGTM 🎈 but fix some minor changes before merge!

File: _slides/05-montseacciona.md (hunk 1 above)
Comment (montseacciona) [reply _slides/05-montseacciona.md:3]: ok make sense

**Review by unaizorrilla — APPROVED** [review]

**Pull request merged**